pages/new-homepage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import JoinUsSection from '../components/home/sections/JoinUs';
+import CollaborateWithMoney from '../components/new-homepage/CollaborateWithMoneySection';
+import DedicatedTeam from '../components/new-homepage/DedicatedTeamSection';
+import GetToKnowUs from '../components/new-homepage/GetToKnowUsSection';
+import OpenCollectiveIs from '../components/new-homepage/OpenCollectiveIsSection';
+import RaiseMoney from '../components/new-homepage/RaiseMoneySection';
+import TheFutureIsCollective from '../components/new-homepage/TheFutureIsCollectiveSection';
+import Page from '../components/Page';
+
+const menuItems = { pricing: true, howItWorks: true };
+
+const messages = defineMessages({
+  defaultTitle: {
+    id: 'OC.tagline',
+    defaultMessage: 'Make your community sustainable. Collect and spend money transparently.',
+  },
+});
+
+const NewHomePage = () => {
+  const { formatMessage } = useIntl();
+  return (
+    <Page menuItems={menuItems} description={formatMessage(messages.defaultTitle)}>
```

Comment:
Great job on this parent component. You made a very neat design decision in the previous homepage to split each section into its own component and you kept this approach here. It makes it very easy to read and navigate.